gui/test/projectfile/testprojectfile.cpp

```diff
@@ -139,6 +139,14 @@ void TestProjectFile::getAddonFilePath() const
     QCOMPARE(ProjectFile::getAddonFilePath(tempdir.path(), filepath), filepath);
 }
 
+void TestProjectFile::getInlineSuppression() const
+{
+    ProjectFile projectFile;
+    projectFile.setFilename("/some/path/123.cppcheck");
+    projectFile.setInlineSuppression(true);
```

Comment:
you could also test that it's true default.
and that setInlineSuppression(false) will make it false.

Reply:
I want that if a project is created with "New project..." menu option in the GUI then this checkbox should be activated directly.. the user will have to turn it off if it's not wanted.